README.md

```diff
@@ -164,7 +174,7 @@ If you like this project then don't forget to give a star ⭐ on this repository
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank [Zeplin](https://app.zeplin.io/login), for the original [design and inspiring my codebase](https://pedantic-beaver-cb923b.netlify.app/#/)
+I would like to thank [Microverse](https://www.microverse.org/), for the original [design guidelines](<https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)>)
```

Comment:
- [ Optional ] In your "Acknowledgements" section Kindly update credit  to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), instead of the Microverse , you can also mention Microverse as required by the [Creative Commons License](https://creativecommons.org/licenses/) .